core/services/ocr2/plugins/functions/functions_config/config.go

```diff
@@ -1,4 +1,4 @@
-package config
+package functions_config
```

Comment:
Did you have to change the directory?

Reply:
In order to conform to the package structure, I did.  w/o the file path change, protoc was not generating the .pb.go file with the correct package name.